src/test/java/com/metamx/common/StreamUtilsTest.java

```diff
@@ -83,29 +83,4 @@ public void flush() throws IOException
     Assert.assertEquals(4, outputFlushes.get());// 2 closes and 2 manual flushes
     Assert.assertArrayEquals(bytes, byteArrayOutputStream.toByteArray());
   }
-  @Test(expected = IOException.class)
```

Comment:
This was testing something that shouldn't be happening (flush on copyWithTimeout )